### PR TITLE
image-lightbox: set `alt` AND `title` on the images

### LINKED ISSUE #618: image-lightbox: set `alt` AND `title` on the images
https://ampbyexample.com/components/amp-image-lightbox/ needs to add:

```
      alt="Picture of a dog"
      title="Picture of a dog, view in lightbox"
```

 to the image.

The `alt` is for the image and the `title` is for the fact that this a button that opens the image in the lightbox. Both are needed since when in the lightbox, `title` does not apply anymore.

### PR DESCRIPTION
**Pull request by sebastianbenz**
Fixes #618

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@aghassemi PTAL

File: src/20_Components/amp-image-lightbox.html [hunk 1]
```diff
@@ -23,45 +23,59 @@
...
-  <amp-img on="tap:lightbox1"
-    role="button"
-    tabindex="0"
-    aria-describedby="imageDescription"
-    src="/img/Border_Collie.jpg" width="300" height="246"></amp-img>
-  <div id="imageDescription">
-    This is a border collie.
+  <div>
+    <amp-img on="tap:lightbox1"
+             role="button"
```
Comment (aghassemi) [inline src/20_Components/amp-image-lightbox.html]: I would still do `title` and `alt` similar to above here in addition to the `aria-describedby` which is providing additional information ( i.e. that it is a `border collie` dog)

Comment (sebastianbenz) [inline src/20_Components/amp-image-lightbox.html]: done - thanks!

**Comment by sebastianbenz** [comment]
@aghassemi ptal

**Comment by aghassemi** [comment]
LGTM. Thansks

**Comment by sebastianbenz** [comment]
Thanks for the review!

**Pull request merged**